modules/circeDerivation/src/test/scala/evo/derivation/circe/EvoDecoderChecks.scala

```diff
@@ -143,10 +152,7 @@ object CheckData:
 
     val writeJson = s"""{"mode" : "w", "append":true, "bin": true}"""
 
-    case class Dictionary(key: String, value: String, next: Option[Dictionary])
-        derives Config,
-          EvoDecoder,
-          EvoEncoder
+    case class Dictionary(key: String, value: String, next: Option[Dictionary]) derives Config, EvoDecoder, EvoEncoder
```

Comment:
Are you sure, you are using the same scalafmt.cfg?

Reply:
Yep, the same one. Perhaps CLI scalafmt tool does something differently, I don't know